### DISCUSSION TITLE
Rust SDK

### DISCUSSION THREAD

**Discussion by kubaplas** [casper-network/condor-info#16] [Q&A] [answered]
I heard rumors about new Rust SDK for Condor release, additionally I see there's a branch `rustSDK-feat-2.0` in casper-rs-client.

What's the status of this project? Should I wait for the SDK to be finished or use the libs from the client to query the node?

**Comment by melpadden** [comment]
There is definitely a new Rust SDK in progress. @asladeofgreen has more detail on this than I

**Comment by ghost** [comment, marked as answer]
@kubaplas 

1. For the moment I would use libs from the client to query the node.  
2. There is a desire and commitment to write an official rust SDK:
     2.1. The code will be completely decoupled from the client.
     2.2. The repo will be [here](https://github.com/casper-network/casper-rust-sdk).
     2.3. Member of R&D team plus core protocol team will be collaborating upon its development which kicks off properly next week.
     2.4. I would assume that an initial version will be ready by end of July.

**Comment by kubaplas** [comment]
@asladeofgreen 
Will the rust SDK be compatible with wasm? We want to use it for our in-browser casper client.
